cmake/wasi-sdk-sysroot.cmake

```diff
@@ -216,14 +237,19 @@ function(define_libcxx target)
     USES_TERMINAL_BUILD ON
     USES_TERMINAL_INSTALL ON
   )
+endfunction()
+
+function(define_libcxx target)
+  define_libcxx_sub(${target} "" "" "")
+  define_libcxx_sub(${target} "-lto" "-flto=full" "/llvm-lto/${llvm_version}")
```

Comment:
Is this `/llvm-lto/$version` convention something codified by Clang itself? Or something we're making up here in wasi-sdk?

Reply:
This is something that was created/added specifically for the wasm clang driver: https://github.com/llvm/llvm-project/blob/0309709a6786653da7164334c83b09c9f37b943a/clang/lib/Driver/ToolChains/WebAssembly.cpp#L206-L212

See https://reviews.llvm.org/D70677

Reply:
Ah ok! Could that be documented here that the convention is derived from LLVM itself?

Reply:
i added a comment